src/ott/geometry/costs.py

```diff
@@ -304,34 +304,45 @@ def pairwise(self, x: jnp.ndarray, y: jnp.ndarray) -> jnp.ndarray:
   def covariance_fixpoint_iter(
       self,
       covs: jnp.ndarray,
-      lambdas: jnp.ndarray,
-      rtol: float = 1e-2
+      weights: jnp.ndarray,
+      tolerance: float = 1e-4,
+      **kwargs
```

Comment:
This is still missing a type hint.

Reply:
ahh sorry, probably added somewhere else!